Mathlib/Algebra/Lie/Derivation/Basic.lean

```diff
@@ -290,4 +292,31 @@ instance instLieAlgebra : LieAlgebra R (LieDerivation R L L) where
 
 end
 
+section
+
+variable {R L : Type*} [Field R] [LieRing L] [LieAlgebra R L]
+
+/-- The linear embedding of Lie derivations into linear endormophisms. -/
+def toLinearMapLinearMap : (LieDerivation R L L) →ₗ[R] (L →ₗ[R] L) where
+  toFun := toLinearMap
+  map_add' := by intro D1 D2; dsimp
+  map_smul' := by intro D1 D2; dsimp
+
+/-- The linear embedding of Lie derivations into linear endormophisms is injective. -/
+lemma toLinearMapLinearMap_injective : Function.Injective ((@toLinearMapLinearMap R L).toFun) := by
+  dsimp [toLinearMapLinearMap]
+  intro D1 D2 h
+  ext a
+  exact congrFun (congrArg DFunLike.coe h) a
+
+/-- Lie derivations over a field form a free module. -/
+instance instModuleFree : Module.Free R (LieDerivation R L L) :=
+  Module.Free.of_divisionRing R (LieDerivation R L L)
```

Comment:
Does inferInstance work? Any module over a field is free.

Reply:
It does, indeed. Thank you! So I removed this instance. However it seems that the `Module.Finite` one is still required.